test/jdk/javax/swing/JMenuItem/MenuItemTest/MenuItemTestHelper.java

```diff
@@ -168,4 +143,23 @@ private static JMenu createSomeIconsMenu(Icon myIcon, Icon myIcon2) {
 
         return someIcons;
     }
+
+    private record ColoredIcon(Color color, int width,
+                               int height) implements Icon {
```

Comment:
```suggestion
    private record ColoredIcon(Color color, int width, int height)
            implements Icon {
```
This would be better. Otherwise, I'd wrap all the parameters, including `width` because both `width` and `height` are tightly connected.